src/main/java/org/example/models/Reference.java

```diff
@@ -0,0 +1,68 @@
+package org.example.models;
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "reference")
+public class Reference {
```

Comment:
Move date from diagnosis to Reference and change naming accordingly

Reply:
Done